.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,18 @@
-
-
 <!--
 Thank you very much for contributing code or documentation to restic! Please
 fill out the following questions to make it easier for us to review your
 changes.
-
-You do not need to check all the boxes below all at once, feel free to take
-your time and add more commits. If you're done and ready for review, please
-check the last box.
 -->
 
-What does this PR change? What problem does it solve?
------------------------------------------------------
+What changes are made in this PR? What problem does it solve?
+-------------------------------------------------------------
```

Comment:
I'm not sold on this.. To me it changes the focus in the question from what actual effect the changes have, to the actual changes in the code. I think it's better to keep the question strongly focused as it is.

Reply:
I agree.

Reply:
I've reverted those changes.